### ISSUE TITLE
Request with payload > 8MB fails

### ISSUE THREAD

**Issue by marcboeker** [mojombo/ernie#1]
Hi,

the following scenario fails for on debian 5.0, 64-bit with Ruby 1.8.7 and Erlang R12B.

```
    =ERROR REPORT==== 26-Oct-2009::17:01:47 ===
    Error in process <0.91.0> with exit value:         {{badmatch,{error,timed_out,<0.52.0>}},[{ernie_server,unsafe_process_now,2},{ernie_server,process_now,2}]}
```

Client side code

```
    svc = BERTRPC::Service.new('192.168.0.1', 2000)
    puts svc.call.deploy.start('x' * 1024 * 1024 * 8)
```

And on the server side

```
    mod(:deploy) do
      fun(:start) do |payload|
        payload.length
      end
    end
```

Reducing the payload to 'x' \* 1024 \* 1000 \* 8 works for me. ATM i don't know if its an BERT encoding or ernie request processing problem.

I'm trying to get into it and provide more detailed information.
cheers // marc

**Comment by mojombo** [comment]
This is fixed in BERT 1.1.0 and included in the Ernie 1.1.0 release.

**Comment by marcboeker** [comment]
ah cool, thanks. i'll give it a shot.
